content/documentation/for-developers.md

```diff
@@ -5,34 +5,23 @@ type: essay
 abstract: "API docs and Quire repository guide"
 ---
 
-## Repositories
-
-There are four repositories hosted on GitHub that make up Quire, which is currently in closed beta. Please [sign-up for access](https://forms.gle/m1fgZu5BHKhddMrW7).
-
-### `quire-cli`
-
-https://github.com/gettypubs/quire-cli
+Quire is in a limited beta and not yet released as open source software. For a free license to use Quire for your publication projects, and for additional access to support, [please sign up](https://docs.google.com/forms/d/e/1FAIpQLScKOJEq9ivhwizmdazjuhxBII-s-5SUsnerWmyF8VteeeRBhA/viewform)
 
-The command-line interface for Quire. It can be installed on macOS, Windows or Linux. It is written in JavaScript and requires [Node.js 12.18.3 LTS](https://nodejs.org) to run. Commands include `quire preview`, `quire pdf`, `quire epub`, and more. A complete reference can be found in [Quire CLI Commands](/documentation/quire-cli/).
+Quire is centered around the static-site generator [Hugo](https://github.com/gohugoio/hugo). It is written in JavaScript and requires [Node.js LTS](https://nodejs.org/en/) to run.
```

Comment:
I'd say specifically that "Quire’s command-line interface is written in JavaScript ..."